workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/IssueToken.kt

```diff
@@ -72,18 +72,21 @@ object IssueToken {
         @Suspendable
         override fun call(): SignedTransaction {
             // This is the identity which will be used to issue tokens.
-            // We also need a session for the other side.
             val me: Party = ourIdentity
-            val ownerSession = initiateFlow(owner)
-
-            // Notify the recipient that we'll be issuing them a tokens and advise them of anything they must do, e.g.
-            // generate a confidential identity for the issuer or sign up for updates for evolvable tokens.
-            ownerSession.send(TokenIssuanceNotification(anonymous = anonymous))
-
+            var ownerSession : FlowSession? = null
             // This is the recipient of the tokens identity.
-            val owningParty: AbstractParty = if (anonymous) {
-                subFlow(RequestConfidentialIdentity.Initiator(ownerSession)).party.anonymise()
-            } else owner
+            val owningParty: AbstractParty
```

Comment:
maybe this can be a conditional assignment? `val owningParty = if (x) { y() } else { z() }`